src/python/aqueduct_executor/operators/connectors/data/relational.py

```diff
@@ -56,11 +57,48 @@ def delete(self, tables: List[str]) -> List[SavedObjectDelete]:
             )
         return results
 
+    def _map_object_dtype_to_varchar(self, df: pd.DataFrame) -> Dict[str, VARCHAR]:
+        """Given a DataFrame, for each string column (i.e. object dtype), it
+        returns a mapping of column name to the appropriate SQL type VARCHAR(N),
+        where N is large enough for all values in the column. Non-string
+        columns will not be included in the returned map.
```

Comment:
Just to be clear, we could be setting this to 65535 each time, but we're trying to cap it to minimize overallocating right?

Reply:
Yup